client/package.json

```diff
@@ -17,19 +17,20 @@
     "@usedapp/core": "^0.5.4",
     "babel-plugin-styled-components": "^1.13.3",
     "cross-env": "^7.0.3",
-    "doppler": "^1.0.0",
```

Comment:
Noticed this was unused and not present in the lockfile. Removed it as it's unused and a risk for versioning 😅